packages/ui-components/src/Library/index.js

```diff
@@ -80,10 +89,30 @@ const LibraryItemCards = styled.div`
 
 const LibraryComponent = (props) => {
   const [query, setQuery] = useState([]);
+  const [ciceroVersion, setCiceroVersion] = useState([]);
   const [itemTypeFilter, setItemTypeFilter] = useState(
     props.itemTypes.reduce((acc, el) => ({ ...acc, [el.type]: true }), {})
   );
 
+  const distinctCiceroVersions = props.items.reduce((acc, { CiceroVersion }) => {
+    if (!acc[CiceroVersion]) {
+      acc[CiceroVersion] = 1;
+    }
+    return acc;
+  }, {});
+  const distinctOptions = [
+    { key: "", value: "", text: "" },
+    ...Object.keys(distinctCiceroVersions).map((CiceroVersion, index) => ({
+      key: index,
+      value: CiceroVersion,
+      text: CiceroVersion,
```

Comment:
Perhaps we could display some more friendly human text when `CiceroVersion` is `undefined`? How about "Unknown", or "Not specified"?